perf-tests/src/test/scala/Simulations.scala

```diff
@@ -0,0 +1,27 @@
+package perfTests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+object Common {
```

Comment:
[minor] you've got two `Common` objects defined in the same package (in main and test); probably would be better to rename this to sth else, like `SimulationCommon` (together with the file name)